cas/stepcas/issuer.go

```diff
@@ -65,11 +66,11 @@ func validateX5CIssuer(iss *apiv1.CertificateIssuer) error {
 	}
 }
 
-// validateJWKIssuer validates the configuration of jwk issuer.
+// validateJWKIssuer validates the configuration of jwk issuer. If the key is
+// not given, then it will download it from the CA. If the password is not given
+// it will be asked.
```

Comment:
If the password is not set it will be prompted.

Reply:
Fixed in 84018ec